provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -1103,6 +1145,7 @@ func generateSchema(version semver.Version) schema.PackageSpec {
 				},
 			},
 			"eks:index:VpcCniAddon": {
+				IsComponent: true,
```

Comment:
Unrelated to this PR, but I noticed this was incorrectly set and we had `.get` functions for this component.